.github/workflows/commit-checker.yml

```diff
@@ -0,0 +1,21 @@
+name: Commit checker
+
+on:
+  pull_request:
+
+jobs:
+  commit-checker:
+    name: Commit checker
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 4
+
+    - name: Get pull-request commits
+      uses: OpenTTD/actions/checkout-pull-request@v5
+
+    - name: Check commits
+      uses: OpenTTD/OpenTTD-git-hooks@main
```

Comment:
This will probably reject the negative test cases.

Reply:
Hmm yeah, might be a problem.

Reply:
I removed this workflow.